src/test/java/strman/StrmanTest.java

```diff
@@ -1027,4 +1025,11 @@ public void charsCount_shouldReturnCharsCountWhenInputIsAComplexString(){
 
         assertThat(charsCount("-----abbcccCCCBBA-----"), equalTo(expectedOutput));
     }
+    
+    @Test
+    public void testIsBlank() {
+    	assertTrue(isBlank(""));
```

Comment:
Please make them seperate test case. Also, use convention used for naming test methods.